lib/bootstrap_form/helpers/bootstrap.rb

```diff
@@ -59,6 +59,15 @@ def static_control(*args, &block)
         end
       end
 
+      def custom_control(*args, &block)
+        options = args.extract_options!
+        name = args.first
+
+        form_group_builder(name, options) do
+          capture(&block)
+        end
+      end
```

Comment:
`extract_options!` is not needed. I've copied `static_control` to keep the same ergonomy.

`capure` is needed to properly render the block which is defined in the view.